### ISSUE TITLE
Commander loses command on death og disconnect.

### ISSUE THREAD

**Issue by stoffer123** [sethduda/AIC#13]
We need a function to keep the commander synced to the command-module even though he dies or reconnects.

**Comment by AVESUM** [comment]
Agree, our persistence campaign is failing on us due to the commander loosing command.
